guides/v2.2/test/js/test_js-unit.md

```diff
@@ -153,26 +158,32 @@ JsTestDriver
 
 To run the automated JavaScript tests, run the `run_js_tests.php` script inside the PHP interpreter from the command line:
 
-    php <magento2_root_dir>/dev/tests/js/run_js_tests.php
+```bash
+php <magento2_root_dir>/dev/tests/js/run_js_tests.php
+```
 
 Find the test results in individual `.xml` files in the `<magento2_root_dir>/dev/tests/js/test-output` directory.
 
 The output of the PHP command resembles this output:
 
 **JsTestDriver output:**
 
-    $ php <magento2_root_dir>dev/tests/js/run_js_tests.php
-    java -jar C:\Users\mchiocca\lib\JsTestDriver.jar --config C:\git\magento2\dev\tests\js/jsTestDriver.conf --port 9876 --browser "C:\Program Files (x86)\Mozilla Firefox\firefox.exe" --tests all --testOutput C:\git\magento2\dev\tests\js/test-output
-    setting runnermode QUIET
-    ....................................
-    Total 36 tests (Passed: 36; Fails: 0; Errors: 0) (138.00 ms)
-      Firefox 15.0 Windows: Run 36 tests (Passed: 36; Fails: 0; Errors 0) (138.00 ms)
+```bash
+$ php <magento2_root_dir>dev/tests/js/run_js_tests.php
```

Comment:
Wrap this line in a `bash` block and remove the $.

Wrap line 173 in a separate `bash` block.

Wrap lines 174 -177 in a `terminal` block.

Reply:
Done